### ISSUE TITLE
Avatar a11y: Circular background is not visible in system high contrast mode

### ISSUE THREAD

**Issue by behowell** [microsoft/fluentui#21355]
### Environment
OS: Windows 11
Browser: Edge
System high contrast theme: Aquatic

### Issue
When setting the Windows system high contrast mode, there is no background or border on the Avatar when it is displaying initials or an icon:
![image](https://user-images.githubusercontent.com/48106640/150339412-ecbd4d8a-fdee-464d-940a-9e338753b4fd.png)

### Expected
The simulated high contrast theme does show a border, which is what the above should look like:
![image](https://user-images.githubusercontent.com/48106640/150339595-7860aedc-cac5-4fd4-9ce0-d2ccf9685280.png)
